engine/apps/slack/scenarios/notification_delivery.py

```diff
@@ -46,17 +46,17 @@ def process_signal(self, log_record):
                     == UserNotificationPolicyLogRecord.ERROR_NOTIFICATION_PHONE_NUMBER_IS_NOT_VERIFIED
                 ):
                     if log_record.notification_channel == UserNotificationPolicy.NotificationChannel.SMS:
-                        self.post_message_to_channel(
+                        self.post_notification_failed_message_to_channel(
                             f"Failed to send an SMS to {user_verbal_with_mention}. Phone number is not verified",
                             alert_group.slack_message.channel_id,
                         )
                     elif log_record.notification_channel == UserNotificationPolicy.NotificationChannel.PHONE_CALL:
-                        self.post_message_to_channel(
+                        self.post_notification_failed_message_to_channel(
                             f"Failed to call to {user_verbal_with_mention}. Phone number is not verified",
                             alert_group.slack_message.channel_id,
                         )
 
-    def post_message_to_channel(self, text, channel):
+    def post_notification_failed_message_to_channel(self, text, channel):
```

Comment:
I would keep the old name because nothing here says about **failed** message

Reply:
+1 here, this method doesn't care about the contents of the message

Reply:
This method is used only in NotificationDeliveryStep and only to send message about failed notification. One of the problem, which this PR is solving is that different parts of code used slack steps methods, which broke the abstraction. That is why I renamed this method. I want to avoid usage of that method in other places, that this step.

Reply:
I would recommend naming it something like `_post_message_to_channel`. The method itself has nothing to do w/ a failure message, but if the intent is to keep it "private" to the `NotificationDeliveryStep` class, then a method named w/ a preceding underscore would be appropriate imo